07LessonSeven/in-class-project/index.html

```diff
@@ -1,10 +1,55 @@
-<!DOCTYPE html>
-<html>
-
-<head>
-    <title>Template</title>
-</head>
-
-<body></body>
-
-</html>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="stylesheet" href="css/my-style.css">
+    <title>NASA</title>
+
+    <header>
+    <style>
+      h1 {
+        padding: 200px;
+      }
+    </style>
+        <h1>NASA</h1>
+    </header>
+
+</head>
+
+<body>
+
+    <main>
+        <section>
+            <div class="one">
```

Comment:
this section has too little padding on the left side, it should be uniform padding all inside this element. other than that, this looks great!